kolibri/utils/options.py

```diff
@@ -177,13 +182,18 @@ def calculate_thread_pool():
 }
 
 
-def get_logger(KOLIBRI_HOME):
+def __get_logger(KOLIBRI_HOME):
```

Comment:
I don't think this is an appropriate use of [name mangling](https://docs.python.org/2/tutorial/classes.html#private-variables-and-class-local-references) since this isn't a class member.

Perhaps just `_get_logger`?

Reply:
Fixed in 668ad5054c9a33a0fc36e1e8108a0e924f7fe412